contracts/token/README.adoc

```diff
@@ -9,6 +9,7 @@ This set of interfaces, contracts, and utilities are all related to `ERC7984`, a
 - {ERC7984ERC20Wrapper}: Extension of {ERC7984} which wraps an `ERC20` into a confidential token. The wrapper allows for free conversion in both directions at a fixed rate.
 - {ERC7984Freezable}: An extension for {ERC7984}, which allows accounts granted the "freezer" role to freeze and unfreeze tokens.
 - {ERC7984ObserverAccess}: An extension for {ERC7984}, which allows each account to add an observer who is given access to their transfer and balance amounts.
+- {ERC7984Restricted}: An extension of {ERC7984} that implements user account transfer restrictions
```

Comment:
Very minor, we could perhaps standardize our phrasing across doc & inline doc: extension _of_ vs extension _for_ (either form you think correct).